overlay.d/05rhcos/usr/lib/dracut/modules.d/40rhcos-fips/rhcos-fips-finish.service

```diff
@@ -3,7 +3,7 @@ Description=Finish FIPS mode setup
 DefaultDependencies=false
 # we run in the second boot after the one where we set fips=1
 ConditionKernelCommandLine=ignition.firstboot
-ConditionKernelCommandLine=fips
+ConditionKernelCommandLine=fips=1
```

Comment:
I guess...but it's not clear to me that we want to support any setup other than (in Rust-like notation):

- `None` (value not present)
- `Some(fips=1)`

It's not supported to set `fips=0` as far as I know.  So I'd argue perhaps we should continue with `ConditionKernelCommandLine=fips`, but then in the code parse its value and abort if it has any value other than `1`.

Reply:
I put this in just to make sure that if somebody puts something unsupported like `fips=0` (or just `fips`) it won't have any effect here.